modules/stanford_profile_styles/lib/scss/paragraph-types/react_paragraphs/index.scss

```diff
@@ -1,4 +1,5 @@
 @charset "UTF-8";
 @import
 '../../config',
-'multirow';
+'multirow',
+'../wysiwyg/paragraph';
```

Comment:
I don't like adding the paragraph as part of the react paragraphs style sheet. It does indeed add it on every page as expected because of [this addition](https://github.com/SU-SWS/stanford_profile_helper/blob/8.x-1.x/modules/stanford_profile_styles/stanford_profile_styles.module#L19), but I believe we can do better than this so that the styles are only added when a wysiwyg is used. It might be overkill at this time and I'm alright with it for now.

@sherakama what do you think.

Reply:
@pookmish ok. Should we attached it the wysiwyg now rather then waiting? Wouldn't it be easier sooner?

Reply:
Yeah why don't we do that now. What we should also do is wrap the selectors with something specific to only wysiwyg fields. like `su-wysiwyg`. Then we can add a preprocess on the fields that have formatted text to wrap that class and add the style sheet.
I can do the preprocessing if you're unsure how. Just compile these styles into a single css so that they can be included dynamically for formatted texts.

Reply:
Ok. For example, Events, doesn't get the su-wysiwyg class. So I could add .events and so on. But I'm wondering if that would make it hard to maintain. Why didn't Events get that class?

Reply:
No that's what i'm referring to. The field itself would get the `su-wysiwyg` class.  I'll put together a preprocess and commit it to this pr so you can test it out.

Reply:
Ah I see now. On it.

Reply:
I fixed it. The class name is `su-wysiwyg-text`. That's what it was before.  thank you!

Reply:
When I use the fields themselves my idea works as expected. but When the field is used in a pattern that is set to display only the content, the preprocess hook doesn't fire and so the styles wouldn't get applied. That could be a good thing in some cases but might be some confusion.
![image](https://user-images.githubusercontent.com/7185045/97027714-fb50c780-150f-11eb-91a7-8699653d4f43.png)

Reply:
I committed the perprocess hook to add that class to all formatted text fields. See how that works on the other content types and we'll go from there.

Reply:
I've tried this on: basic page, News, People, Event and Event Series. It works like I want on all of them. I couldn't find a page example of your screenshot.